projects/packages/forms/src/modules/file-field/view.js

```diff
@@ -36,6 +95,9 @@ const addFileToContext = file => {
 	const reader = new FileReader();
 	reader.readAsDataURL( file );
 
+	const { ref } = getElement();
+	clearInputError( ref, { hasInsetLabel: isInlineForm( ref ) } );
```

Comment:
Not a big deal, but `isInlineForm` could be derived state.

```suggestion
	clearInputError( ref, { hasInsetLabel: state.isInlineForm } );
```

```js
store( '...', {
  state: {
    get isInlineForm() {
      const { ref } = getElement();
      const form = ref.closest( '.wp-block-jetpack-contact-form' );
    	return form && form.classList.contains( 'is-style-outlined' ) ) ||
    		form.classList.contains( 'is-style-animated' );
    }
  }
});
```

Reply:
Good call.